common/src/main/java/org/apache/celeborn/common/protocol/PartitionLocation.java

```diff
@@ -59,6 +59,8 @@ public static String getFileName(String uniqueId, Mode mode) {
   private int pushPort;
   private int fetchPort;
   private int replicatePort;
+  private int internalPort;
+  private int securedPort;
```

Comment:
Can we add a note on when which port is to be used/is valid ?
Either here, or as javadoc to the `get*Port` methods below ?